test/test-worktree.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+. "test/testlib.sh"
+
+# git 2.5 or higher only
+compare_version $(git version | cut -d" " -f3) "2.5.0"
+if [[ $? != $VERSION_LOWER ]]; then
+  begin_test "git worktree"
```

Comment:
Doh, I didn't do that because I thought 'I don't want to exit the whole test suite' but of course exit  just affects the current script.